app/views/posts/show.html.erb

```diff
@@ -1,3 +1,25 @@
 <main id="content">
-  <%= link_to  @post.title, @post.url %>
-</main>  
+  <br>
+ <%= link_to @post.title, @post.url %>
+  <br>
+    Posted by <%= link_to @post.user.name, @post.user %>
+    <%= time_ago_in_words(@post.created_at) %> ago.
+  <br>
+  <br>
+  <%= form_for @comment, url: comments_path do |f| %>
+    <%= render 'shared/comment_errors', comment: @comment %>
+    <%= f.text_area :content, placeholder: "Add a comment", cols: 60, rows: 10  %>
+    <%= f.hidden_field :post_id, value: @post.id %>
+    <% if logged_in? %>
+      <%= f.hidden_field :user_id, value: current_user.id %>
+    <% end %>
+    <%= f.submit "Comment" %>
+  <% end %>
```

Comment:
I could certainly render a shared form for both the posts/show.html.erb and the comments/_comment.html.erb partial because the basic functionality is the same. I guess I'll just have to pass in an additional parameter to the render call for the comments view in order to contain the comment_id.